sdk/python/jobs/automl-standalone-jobs/automl-distributed-classification-task-creditcard/classification-distributed-lightgbm.ipynb

```diff
@@ -0,0 +1,1472 @@
+    "# my_training_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/my_training_mltable\")\n",
+    "# my_validation_data_input = Input(type=AssetTypes.MLTABLE, path=\"azureml://datastores/workspaceblobstore/paths/my_validation_mltable\")"
+   ],
+   "outputs": [],
+   "execution_count": null,
+   "metadata": {
+    "jupyter": {
+     "source_hidden": false,
+     "outputs_hidden": false
+    },
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    },
+    "gather": {
+     "logged": 1684844838777
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "For documentation on creating your own MLTable assets for jobs beyond this notebook:\r\n",
+    "- https://learn.microsoft.com/en-us/azure/machine-learning/reference-yaml-mltable details how to write MLTable YAMLs (required for each MLTable asset).\r\n",
+    "- https://learn.microsoft.com/en-us/azure/machine-learning/how-to-create-data-assets?tabs=Python-SDK covers how to work with them in the v2 CLI/SDK."
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## 3. Create or Attach existing AmlCompute\r\n",
+    "\r\n",
+    "[Azure Machine Learning Compute](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-set-up-training-targets#amlcompute) is a managed-compute infrastructure that allows the user to easily create a single or multi-node compute. In this tutorial, you will create and an AmlCompute cluster as your training compute resource.\r\n",
+    "\r\n",
+    "<b>Creation of AmlCompute takes approximately 5 minutes.</b>\r\n",
+    "\r\n",
+    "If the AmlCompute with that name is already in your workspace this code will skip the creation process.\r\n",
+    "As with other Azure services, there are limits on certain resources (e.g. AmlCompute) associated with the Azure Machine Learning service. Please read [this article](https://docs.microsoft.com/en-us/azure/machine-learning/service/how-to-manage-quotas) on the default limits and how to request more quota."
+   ],
+   "metadata": {
+    "nteract": {
+     "transient": {
+      "deleting": false
+     }
+    }
+   }
+  },
+  {
+   "cell_type": "code",
+   "source": [
+    "from azure.core.exceptions import ResourceNotFoundError\n",
+    "from azure.ai.ml.entities import AmlCompute\n",
+    "\n",
+    "cluster_name = \"automl-distributed-cluster\"\n",
+    "\n",
+    "try:\n",
+    "    # Retrieve an already attached Azure Machine Learning Compute.\n",
+    "    compute = ml_client.compute.get(cluster_name)\n",
+    "except ResourceNotFoundError as e:\n",
+    "    compute = AmlCompute(\n",
+    "        name=cluster_name,\n",
+    "        size=\"STANDARD_DS12_V2\",\n",
+    "        type=\"amlcompute\",\n",
+    "        min_instances=4,\n",
```

Comment:
Let's use min_instances as zero. We do this in all notebooks.

Reply:
I have set it to 4 with the description that min_instances need to be set to 4 for distributed training. Thank you.